src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java

```diff
@@ -330,7 +330,7 @@ public List<Movement> newMultipleChargingMovements(List<Movement> movements, Str
 				errors.add(new OHExceptionMessage(
 								mov.getMedical() != null ? mov.getMedical().getDescription()
 												: MessageBundle.getMessage("angal.medicalstock.nodescription.txt")));
-				throw new OHDataValidationException(errors);
+				throw new OHDataValidationException(errors.get(0));
```

Comment:
Revert change that was breaking many tests
```suggestion
				throw new OHDataValidationException(errors);
```